base/applications/mspaint/dib.cpp

```diff
@@ -519,3 +526,89 @@ HBITMAP BitmapFromHEMF(HENHMETAFILE hEMF)
 
     return hbm;
 }
+
+BOOL IsBitmapBlackAndWhite(HBITMAP hbm)
+{
+    BITMAP bm;
+    if (!::GetObjectW(hbm, sizeof(bm), &bm))
+        return FALSE;
+
+    if (bm.bmBitsPixel == 1)
+        return TRUE;
+
+    BITMAPINFOEX bmi;
+    ZeroMemory(&bmi, sizeof(bmi));
+    bmi.bmiHeader.biSize = sizeof(BITMAPINFOHEADER);
```

Comment:
It should be possible to do:
```suggestion
    bmi.bmiHeader.biSize = sizeof(bmi.bmiHeader);
```